src/wwwroot/template-editor/adapterService.js

```diff
@@ -9,13 +12,15 @@
   adapterService.$inject = [
     '$http',
     '$q',
-    '$window'
+    '$window',
+    'RELAY_CONFIG'
```

Comment:
is it not necessary to indicate a kind of namespace? or something like that?

Reply:
Not, due to constants providers works as the same as a service, it is enabled to the entire app once the module that define it is included in the angular app scope.

Reply:
In that case, there may not be any advantage in having _dopplerRelay_ module and this could be a better solution: https://github.com/MakingSense/MSEditor/blob/6a2a90d6f442184d93fabdb539540819c82a303f/refactor/gulpfile.js#L481-L488

Reply:
I do not understand, I will call you.

Reply:
I will add this approach like a TODO task